### ISSUE TITLE
In Game Time In Between Segments

### ISSUE THREAD

**Issue by Isntmexion** [Minecraft-Java-Edition-Speedrunning/rules#50]
[Old Issue 54]
Reported by Loth
Thread: https://discord.com/channels/83066801105145856/1252088803494072330/1252199381470281750
Affected Rules/Sections: A.5.10) Time spent between segments of a run does not contribute to in-game time.

This rule is redundant, as you cannot be unpaused in the world between segments.

**Comment by r0hkx** [comment]
the full text of the rule is A.5.10) If a run is segmented, then time spent between the segments of the run will not contribute to in-game time *or the real time of the run.* 

emphasis mine. this is still an important part of the rule, no?

also, IGT on speedrun.com can refer to a retimed time, not just pure IGT.

**Comment by Isntmexion** [comment]
Good catch! This is a copy of #89 which has been resolved. This is the old wording of the rule.
